test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
we should have many new GPU-enabled LSHW fixtures to test this functionality. Can you commit a bunch of sample LSHWs from different hw to have a more robust test suite?